test_fixtures/align_baseline_multiline_column.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <script src="../scripts/gentest/test_helper.js"></script>
+  <link rel="stylesheet" type="text/css" href="../scripts/gentest/test_base_style.css">
+  <title>
+    Test description
+  </title>
+<head/>
+<body>
+
+<!-- Broken after Chrome behavior change -->
```

Comment:
Does this mean that the test generated by the gentest will be incorrect? What is the correct course of action in these cases? 

Do we disable the test or do we keep it as chrome's new behaviour is what we are targeting? 
I do suspect once you throw multiple browsers into the mix and compare them the picture becomes a lot more complicated.

Reply:
Actually these comments come directly from the yoga repo (where these tests are commented out). We actually pass these tests just fine (and https://github.com/DioxusLabs/taffy/pull/325 fixes the baseline alignment tests we weren't already passing). And in the case where we don't, our policy (at least for now) is just to update to the latest "web compatible" behaviour. On the other hand Yoga doesn't want to change their exisiting behaviour because it's used in react native which has a huge install base whose UIs might all end up broken by such a change.

I've removed the comments from our codebase.